core-it-suite/src/test/java/org/apache/maven/it/MavenITmng4262MakeLikeReactorDottedPathTest.java

```diff
@@ -64,7 +64,6 @@ public void testitMakeRoot()
         verifier.resetStreams();
 
         verifier.assertFilePresent( "target/touch.txt" );
-        verifier.assertFileNotPresent( "../sub-a/target/touch.txt" );
```

Comment:
Why is this one gone?

Reply:
This test used to verify that if you execute `mvn <goal> -pl .` that root is built (line 66) and that the subproject is not built (line 67).
The change in MNG-6981 changed the behavior of `-pl`. First it didn't build the target project's children, now it does. 

I decided to remove the assertion on line 67, since it is in my opinion an unnecessary assertion which is not related to unit test. Also by removing the assertion we make sure that this test will work pre and post MNG-6981. Alternatively we could duplicate the test class with different version restrictions. What do you think?

Reply:
Does the test still pass with 3.6.3? The are assertions which can be scoped with version ranges. Maybe we could utilize them?

Reply:
I have not found assertions with version scopes, but I've found the method `matchesVersionString` which I applied. I have verified it by running Maven 3.6.3 and the MNG-6981 PR against the ITs. WDYT?

Reply:
That's exactly what I was referring to!

Reply:
The general rule is that you should never adjust the test itself. If the behavior changes, adjust the Maven versionsRange and make a new integrationtest for it (with the new behavior)

Reply:
Ah, that's right, I forgot about that. Too bad we'll duplicate the tests then, but it makes sense to have the old tests 'immutable'. I took your advice, @rfscholte , this is what you meant, right?

Reply:
Almost correct, you should only adjust the value in the constructor, this one should be

    super( "(,3.7.0)" );

Reply:
Oh.. my `git push` didn't get through... I've pushed the changes now. Please take another look..